docs/governance/templates/release.md

```diff
@@ -47,15 +52,9 @@ _This software is currently alpha, and subject to change. Not to be used in prod
 Installation
 ------------
 
-To deploy Open Match in your Kubernetes cluster run the following commands:
+* Follow [Open Match Installation Guide](https://open-match.dev/site/docs/installation/) to setup Open Match in your cluster.
 
-```bash
-# Grant yourself cluster-admin permissions so that you can deploy service accounts.
-kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=$(YOUR_KUBERNETES_USER_NAME)
-# Place all Open Match components in their own namespace.
-kubectl create namespace open-match
-# Install Open Match services.
-kubectl apply -f https://github.com/googleforgames/open-match/releases/download/v{version}/01-open-match-core.yaml --namespace open-match
-# Install the demo.
-kubectl apply -f https://github.com/googleforgames/open-match/releases/download/v{version}/02-open-match-demo.yaml --namespace open-match
-```
+API Definitions
+------------
+
+Available both in [SwaggerUI](https://open-match.dev/site/swaggerui/index.html) and [API references](https://open-match.dev/site/docs/reference/api/).
```

Comment:
We want users to use proto over rest, can be emphasize this by clarifying which reference is for which (rest vs grpc), and putting grpc first?